testsuite/tests/singlecluster/authorino/wristband/conftest.py

```diff
@@ -90,6 +74,31 @@ def wristband_hostname(exposer, gateway, blame):
     return exposer.expose_hostname(blame("route"), gateway)
 
 
+@pytest.fixture(scope="module")
+def route_dictionary():
+    """Create dictionary with wristband specific route match for EnvoyConfig."""
+    route_dictionary = {
+        "match": {"prefix": "/auth"},
+        "directResponse": {"status": 200},
+        "response_headers_to_add": [
+            {
+                "header": {
+                    "key": "wristband-token",
+                    "value": '%DYNAMIC_METADATA(["envoy.filters.http.ext_authz", "wristband"])%',
+                }
+            }
+        ],
+    }
+    return route_dictionary
+
+
+@pytest.fixture(scope="module")
+def route(route, route_dictionary):
```

Comment:
Isn't it enough to just edit the envoy config once? Both these routes point to the same gateway, so I'd think they have a single config for both